compiler/test/fail_compilation/throwexp.d

```diff
@@ -0,0 +1,11 @@
+/+ TEST_OUTPUT:
+---
+fail_compilation/throwexp.d(11): Error: to `throw` an expression it must be a non-null `class` reference
+fail_compilation/throwexp.d(11):        `ret()` has type `object.Exception`
```

Comment:
Because this supplemental error is not helpful here

Reply:
will make it moar selective